code/src/components/News/News.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import Headline from '../Shared/Headline/Headline'
+import Album from './Album/Album'
+import data from '../../data.json'
```

Comment:
Nothing wrong with relative imports, but just fyi, the project is set up so you can do absolute imports (from 'src'). So these could be:

```
import Headline from 'Shared/Headline/Headline'
import Album from 'components/Album/Album'
import data from 'data.json'
```

It just makes it nice not to have to think so much about folder structure.